### DISCUSSION TITLE
mute and unmute with two players

### DISCUSSION THREAD

**Discussion by nayotorres** [PierfrancescoSoffritti/android-youtube-player#1089] [General]
Hello, good afternoon, I am doing an implementation, and I have the following problem.

I am in a Picture-in-picture view, the second player is muted, when I change the audio, I put the first player on mute and the second player unMute, but for some reason the first player stays on pause, do you know why this happens to me?

**Comment by PierfrancescoSoffritti** [comment]
Android doesn't allow to players to play at the same time. So you probably need to invoke play to resume the video.
